environment.yml

```diff
@@ -1,6 +1,29 @@
 name: tmprss2
+
 channels:
   - conda-forge
+  - pytorch
+  - rdkit
+
 dependencies:
-  - python=3
+  - flask>=1.1.2
+  - gunicorn>=20.0.4
+  - hyperopt>=0.2.3
+  - matplotlib>=3.1.3
+  - numpy>=1.18.1
+  - pip>=20.0.2
+  - pytorch>=1.4.0
+  - rdkit>=2020.03.1.0
+  - scikit-learn>=0.22.2.post1
+  - scipy>=1.4.1
+  - tensorboardX>=2.0
+  - tensorflow>=1.13.1
+  - torchvision>=0.5.0
+  - tqdm>=4.45.0
+  - molvs
+  - ipykernel
+  - seaborn
   - pandas
+  - pip:
+    - typed-argument-parser>=1.4.3
+    - git+https://github.com/bp-kelley/descriptastorus
```

Comment:
The reason I didn't just physically copy/paste chemprop's .yml into ours was because in theory, chemprop is a fluid project, and they might change their dependencies as they change their code.  So directly using their .yml means one less thing to maintain.

In general, my principle was to try and keep our project and chemprop separated.  If we decide we want to go in and start editing their code, though, this all goes out the window anyways.  I see reason behind both ways of thinking, so I'll leave the choice up to you.

Reply:
I see the case, however, it also happens to be that our repo is currently tracking a specific commit of the chemprop repo. I understand that you have had troubles with installing chemprop and this pip install -e was your solution to overcome these issues. Chemprop will likely not change in the scope of our project and also if it were, our repo would still be linked to a the tag "e9...". Already the installation process is somewhat difficult for someone not too familiar with GitHub (as the download button in "Clone or download" does not work for due to submodule), so I think it is best to keep it as simple/easy to get it done.

Reply:
Like if you foresee coming back to this repo month after month, updating your submodule link to the latest, I see your point, but maybe for now we keep it simpler/not-as-good-practice :)